### PR TITLE
Feature: Add to Calendar #5072

### PR DESCRIPTION
**Pull request by akash2408**
Fixes #

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/public/social-links.js [hunk 1]
```diff
@@ -9,5 +10,13 @@ export default Component.extend({
 
   twitterLink: computed('socialLinks.[]', function() {
     return this.socialLinks.findBy('isTwitter', true);
+  }),
+   calender:computed('eventtime','eventtimeends','eventname','eventlocation',function(){
+   let startparams=this.get('eventtime');
+   let endparams=this.get('eventtimeends');
```
Comment (niranjan94) [inline app/components/public/social-links.js:16]: Codacy found an issue: [Operator '=' must be spaced.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 17
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364&bid=20749959)

File: app/components/public/social-links.js [hunk 2]
```diff
@@ -9,5 +10,13 @@ export default Component.extend({
...
+   const starttime=moment(startparams).tz("Asia/Kolkata").utc().format('YYYYMMDD[T]HHmmSS[Z]');
+   const endtime=moment(endparams).tz("Asia/Kolkata").utc().format('YYYYMMDD[T]HHmmSS[Z]');
```
Comment (niranjan94) [inline app/components/public/social-links.js:18]: Codacy found an issue: [Expected indentation of 4 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js [hunk 3]
```diff
@@ -9,5 +10,13 @@ export default Component.extend({
 
   twitterLink: computed('socialLinks.[]', function() {
     return this.socialLinks.findBy('isTwitter', true);
+  }),
+   calender:computed('eventtime','eventtimeends','eventname','eventlocation',function(){
+   let startparams=this.get('eventtime');
```
Comment (niranjan94) [inline app/components/public/social-links.js:15]: Codacy found an issue: ['startparams' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js [hunk 4]
```diff
@@ -9,5 +10,13 @@ export default Component.extend({
 
   twitterLink: computed('socialLinks.[]', function() {
     return this.socialLinks.findBy('isTwitter', true);
+  }),
+   calender:computed('eventtime','eventtimeends','eventname','eventlocation',function(){
```
Comment (niranjan94) [inline app/components/public/social-links.js:14]: Codacy found an issue: [A space is required after ','.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js [hunk 5]
```diff
@@ -9,5 +10,13 @@ export default Component.extend({
...
+   let startparams=this.get('eventtime');
+   let endparams=this.get('eventtimeends');
+   const starttime=moment(startparams).tz("Asia/Kolkata").utc().format('YYYYMMDD[T]HHmmSS[Z]');
```
Comment (niranjan94) [inline app/components/public/social-links.js:17]: Codacy found an issue: [Strings must use singlequote.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 4 above)
Comment (niranjan94) [inline app/components/public/social-links.js:14]: Codacy found an issue: [Missing space before opening brace.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 3 above)
Comment (niranjan94) [inline app/components/public/social-links.js:15]: Codacy found an issue: [Operator '=' must be spaced.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 1 above)
Comment (niranjan94) [inline app/components/public/social-links.js:16]: Codacy found an issue: [Expected indentation of 4 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 5 above)
Comment (niranjan94) [inline app/components/public/social-links.js:17]: Codacy found an issue: [Expected indentation of 4 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

Comment (niranjan94) [inline app/components/public/social-links.js:17]: Codacy found an issue: [Operator '=' must be spaced.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 2 above)
Comment (niranjan94) [inline app/components/public/social-links.js:18]: Codacy found an issue: [Strings must use singlequote.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

Comment (niranjan94) [inline app/components/public/social-links.js:18]: Codacy found an issue: [Operator '=' must be spaced.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 3 above)
Comment (niranjan94) [inline app/components/public/social-links.js:15]: Codacy found an issue: [Expected indentation of 4 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js [hunk 6]
```diff
@@ -9,5 +10,13 @@ export default Component.extend({
...
+   const endtime=moment(endparams).tz("Asia/Kolkata").utc().format('YYYYMMDD[T]HHmmSS[Z]');
+   return `https://calendar.google.com/calendar/u/0/r/eventedit?dates=${starttime}/${endtime}&text=${this.get('eventname')}&location=${this.get('eventlocation')}&sf=true`;
```
Comment (niranjan94) [inline app/components/public/social-links.js:19]: Codacy found an issue: [Expected indentation of 4 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 4 above)
Comment (niranjan94) [inline app/components/public/social-links.js:14]: Codacy found an issue: [Expected indentation of 2 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 1 above)
Comment (niranjan94) [inline app/components/public/social-links.js:16]: Codacy found an issue: ['endparams' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

File: app/components/public/social-links.js (hunk 4 above)
Comment (niranjan94) [inline app/components/public/social-links.js:14]: Codacy found an issue: [Missing space before value for key 'calender'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

Comment (iamareebjamal) [inline app/components/public/social-links.js:14]: None of these properties exist

**Comment by iamareebjamal** [comment]
You've made random changes and not tested the PR, please add a screenshot/video of the feature working

**Comment by akash2408** [comment]
ohk sure i will attach the screenshot

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364&bid=20749959)

File: app/components/public/social-links.js [hunk 7]
```diff
@@ -9,5 +10,14 @@ export default Component.extend({
 
   twitterLink: computed('socialLinks.[]', function() {
     return this.socialLinks.findBy('isTwitter', true);
+  }),
+
+  calender: computed( function() {
```
Comment (niranjan94) [inline app/components/public/social-links.js:15]: Codacy found an issue: [There should be no space after this paren.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)

**Pull request closed without merging**